content/articles/boosting-algorithms-part-1/index.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: engineering-education
+status:
+published:
+url:
+title: Boosting Algorithms (Part 1)
+description: Boosting is an ensemble method and is a statistical technique that is used to enhance the performance of a machine learning model by converting weak learners to strong learners.
+author: prashanth-saravanan
+date:
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/boosting-algorithms-part-1/hero.jpg
+    alt: Boosting Algorithms
+---
+This article is aimed at refreshing the reader of their knowledge of boosting algorithms, how different they are from the existing performance-enhancing algorithms, and discusses the existing boosting models.
+<!--more-->
+Boosting has quickly risen to be one of the most sought-after techniques to improve the performance of models in machine learning. There has been an exponential rise in the usage of boosting algorithms in the world of Kaggle. One can hardly pick a model at the top 20 of any competition that hasn't used a boosting algorithm.
+
+### Table of Contents
+1. [Introduction](#introduction)
+2. [Pre-Requisites](#pre-requisites)
+3. [Bagging vs Boosting](#bagging-vs-boosting)
+4. [Data Preprocessing](#data-preprocessing)
+4. [AdaBoost](#adaboost)
+5. [AdaBoost Implementation in Python](#adaboost-implementation-in-python)
+6. [XGBoost](#xgboost)
+7. [XGBoost Implementation in Python](xgboost-implementation-in-python)
+8. [Additional Resources](#additional-resources)
+
+### Introduction
+The first-ever boosting algorithm called AdaBoost (Adaptive Boosting) was introduced to the world by Freund and Schapire. [Statisticians](https://web.stanford.edu/~hastie/Papers/buehlmann.pdf) have come forward, laying proofs and theorems as to how this gradient descent algorithm in function space, based on statistical estimations and numerical optimizations has proven to be very competitive in terms of prediction accuracy.
+
+AdaBoost was described as a "stagewise, additive modeling", where "additive" didn't mean a model fit added by covariates, but meant a linear combination of estimators. An ensemble technique almost always confused with bagging, boosting can be considered as extensively regularizing a model through model iterations. 
```

Comment:
This intro looks a little technical. Beginners would find it difficult to understand.

Instead of directly jumping into Adaboost, you could probably talk a little about Ensemble methods, and boosting.